client/src/js/actions/AppActions.js

```diff
@@ -3,173 +3,348 @@ import AppConstants from '../constants/AppConstants';
-    AppDispatcher.handleViewAction({
-      actionType: AppConstants.RECEIVE_ALLUSERS_NUMBER,
-      number
-    });
-  },
 
+  /**
+   * @description describes an action that makes
+   * API call to the server for a post request to get all users
+   * who have seen a message.
+   *
+   * @param { Object } user
+   *
+   * @returns { Object } returns an object containing list of users
+   */
   seenMessage(user) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.SEEN_MESSAGE,
       user
     });
   },
 
-  receiveSeenUsers(users) {
-    AppDispatcher.handleViewAction({
-      actionType: AppConstants.RECEIVE_SEEN_USERS,
-      users
-    });
-  },
-
+   /**
+   * @description describes an action that makes
+   *  API call to the server for a post request to save a user's group
+   *
+   * @param { Object } group
+   *
+   * @returns { Object } returns notification message
+   */
   saveGroup(group) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.SAVE_GROUP,
       group
     });
   },
 
+     /**
+   * @description describes an action that makes
+   * an API call to the server for a get request
+   * to get all the groups that a user belongs to.
+   *
+   * @param { Object } userName
+   *
+   * @returns { void }
+   */
   getGroups(userName) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.GET_GROUPS,
       userName
     });
   },
 
-  receiveGroup(groups) {
-    AppDispatcher.handleViewAction({
-      actionType: AppConstants.RECEIVE_GROUP,
-      groups
-    });
-  },
 
+  /**
+   * @description describes an action that
+   * receive all user's notification.
+   *
+   * @param { Object } notification
+   *
+   * @returns { Object } returns an object containing user's notification
+   */
   receiveNotification(notification) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.RECEIVE_NOTIFICATION,
       notification
     });
   },
 
+  /**
+   * @description describes an action that makes
+   * receive all groups that a user belong to.
+   *
+   * @param { Object } groups
+   *
+   * @returns { Object } returns an object containing user's group
+   */
   receiveGroups(groups) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.RECEIVE_GROUPS,
       groups
     });
   },
 
+  /**
+   * @description describes an action that makes
+   * an API call to the server for a post request
+   * to save a user into a group.
+   *
+   * @param { Object } addUser
+   *
+   * @returns { Object } returns a notification message
+   */
   saveGroupUser(addUser) {
     AppDispatcher.handleViewAction({
       actionType: AppConstants.SAVE_GROUP_USER,
       addUser
     });
   },
 
-  saveMessage(message) {
+  /**
+   * @description describes an action that makes
+   * an API call to the server for a post request
+   * to save a message.
+   *
+   * @param { Object } message
+   *
+   * @returns { void }
+   */
+  postMessage(message) {
```

Comment:
IMO, the function name should reflect the action name i.e saveMessage for SAVE_MESSAGE for postMessage for POST_MESSAGE